lib/dotcom_web/components/trip_planner/itinerary_group.ex

```diff
@@ -3,6 +3,8 @@ defmodule DotcomWeb.Components.TripPlanner.ItineraryGroup do
   A component to render an itinerary group.
   """
 
+  import DotcomWeb.Components.{ModeIcons, Svg}
+  alias Dotcom.TripPlan.{TransitDetail, PersonalDetail}
   use DotcomWeb, :component
```

Comment:
We try to order things `use`, `require`, `import`, `alias`. Ordered by impact. Also, an empty line in between the different include types and everything alphabetized. Really helps when includes become large.

```
use DotcomWeb, :component

import DotcomWeb.Components.{ModeIcons, Svg}

alias Dotcom.TripPlan.{PersonalDetail, TransitDetail}
```

Reply:
Ahh - thanks for this! I remember a conversation somewhere else (slack?) where we were talking this through - this is a helpful example of that.

Reply:
Heh - side benefit - I noticed two import statements that I was able to combine.